lm_eval/tasks/__init__.py

```diff
@@ -188,7 +188,11 @@
     **wmt.create_year_tasks(wmt.WMT14_TASKS),
     # DiaBLa
     "diabla": diabla.DiaBLa,
-    # piaf
+   # add_xquad 
+    "xquad.en": xquad.xquad_en,
+    "xquad.ar": xquad.xquad_ar,
```

Comment:
Replace the dots `.`  in the keys with underscores to match the formatting of the other task names.